packages/viz/src/index.d.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-unused-vars */
+import DataUtil from './types/utils/data'
+import { CbgDateTrace } from 'dumb/src/models/cbg-date-trace.model'
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Is this not creating a circular dependency? With `dumb` importing actions from `viz`, and `viz` now importing models from `dumb`...

(maybe we can say it's OK while it doesn't break anything, and the redux exports are only temporary and should be removed soon 🧹 )